external/wypopovercontroller/.gitignore

```diff
@@ -0,0 +1,18 @@
+# Xcode
```

Comment:
Why are these files committed? Shouldn't `external/wypopovercontroller` be deleted? If not, it should still be consumed as submodule.

Reply:
I had deleted `external/wypopovercontroller` but SmartSync wouldn't compile without it.  I added it back to avoid modifying the code this late, but I can take a closer look.

Reply:
Looks like the submodule got removed and the files got added back incorrectly.

Reply:
Yeah, sorry I was going to write that in comments but I just got pulled into a call.  wypopovercontroller should have been added back as a submodule, that is a mistake.  Shared also got added back as files to avoid creating another PR to create a branch over there.  That is the correct way to do it, but I don't know if its worth it if this branch just get deleted when we release 7.0 proper.

Reply:
Let's do the submodule bit correctly. I'm not sure we want zombie files just hanging around with no source.